README.md

```diff
@@ -1,4 +1,4 @@
-# ![recheck logo](https://user-images.githubusercontent.com/1871610/41766965-b69d46a2-7608-11e8-97b4-c6b0f047d455.png)
+ #### <a href="http://retest.dev"><img src="https://assets.retest.org/retest/ci/logos/recheck-screen.svg" width="300"/></a> fully functional open source testing tool
```

Comment:
Header level change intended? And shouldn't it be "fully-functional open-source testing tool" (compound adjectives)? Also website says "fully functional free software testing API"?